jenkins/bootstrap.py

```diff
@@ -374,7 +424,7 @@ def node():
     return os.environ[NODE_ENV]
 
 
-def find_version():
+def find_version(call):
```

Comment:
can you explain how `call` is being used here inside the function? Do you need some implementation as above in  `checkout`?